web/packages/teleport/src/DeviceTrust/DeviceTrustLocked.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
```

Comment:
Don't forget license header.

Reply:
Oh yes! added in https://github.com/gravitational/teleport/pull/30249/commits/eb4015be958e231bf29722bb43d44c58dd5ea34f